halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/travis/TravisEditMasterCommand.java

```diff
@@ -47,11 +47,18 @@ protected String getCiName() {
   )
   public String githubToken;
 
+  @Parameter(
+    names = "--number-of-repositories",
+    description = TravisCommandProperties.NUMBER_OF_REPOSITORIES_DESCRIPTION
+  )
+  public int numberOfRepositories = 25;
```

Comment:
If you provide a default here, it will be set everytime someone runs `edit` on this command, even if they don't intend to.

Reply:
Thank you, I will fix this.